integration/simulation/devnetwork/sim_l1.go

```diff
@@ -0,0 +1,15 @@
+package devnetwork
+
+import (
+	"github.com/obscuronet/go-obscuro/go/ethadapter"
+	"github.com/obscuronet/go-obscuro/integration/gethnetwork"
+)
+
+type L1Network struct {
+	clients     []ethadapter.EthClient
+	gethNetwork *gethnetwork.GethNetwork
```

Comment:
Does this mean this setup doesn't support the fully in-memory mocked l1 nodes?

Reply:
Yeah v good point, I hadn't thought about it, just considered that the in-mem sim would live on its current form but better to make this thing flexible so it can be used to refactor them both.